lib/parse.js

```diff
@@ -165,7 +166,15 @@ CSSOM.parse = function parse(token) {
 				i += "container".length;
 				buffer = "";
 				break;
-			} else if (token.indexOf("@supports", i) === i) {
+			} else if (token.indexOf("@layer", i) === i) {
+				state = "layerBlock"
+				layerBlockRule = new CSSOM.CSSLayerBlockRule();
+				layerBlockRule.__starts = i;
+				i += "layer".length;
+				buffer = "";
+				break;
+			} 
+			else if (token.indexOf("@supports", i) === i) {
```

Comment:
```suggestion
			}  else if (token.indexOf("@supports", i) === i) {
```

other style surrounding it has this style